src/otx/v2/adapters/torch/mmengine/mmaction/modules/models/backbones/movinet.py

```diff
@@ -0,0 +1,780 @@
+            self.expand = ConvBlock3D(
+                in_planes=cfg.input_channels,
+                out_planes=cfg.expanded_channels,
+                kernel_size=(1, 1, 1),
+                padding=(0, 0, 0),
+                conv_type=conv_type,
+                tf_like=tf_like,
+                norm_layer=norm_layer,
+                activation_layer=activation_layer,
+            )
+        self.deep = ConvBlock3D(
+            in_planes=cfg.expanded_channels,
+            out_planes=cfg.expanded_channels,
+            kernel_size=cfg.kernel_size,
+            padding=cfg.padding,
+            stride=cfg.stride,
+            groups=cfg.expanded_channels,
+            conv_type=conv_type,
+            tf_like=tf_like,
+            norm_layer=norm_layer,
+            activation_layer=activation_layer,
+        )
+        # pylint: disable=invalid-name
+        self.se = SqueezeExcitation(
+            cfg.expanded_channels,
+            activation_1=activation_layer,
+            activation_2=(nn.Sigmoid if conv_type == "3d" else nn.Hardsigmoid),
+            conv_type=conv_type,
+        )
+        self.project = ConvBlock3D(
+            cfg.expanded_channels,
+            cfg.out_channels,
+            kernel_size=(1, 1, 1),
+            padding=(0, 0, 0),
+            conv_type=conv_type,
+            tf_like=tf_like,
+            norm_layer=norm_layer,
+            activation_layer=nn.Identity,
+        )
+
+        if not (cfg.stride == (1, 1, 1) and cfg.input_channels == cfg.out_channels):
+            if cfg.stride != (1, 1, 1):
+                if tf_like:
+                    layers.append(TFAvgPool3D())
+                else:
+                    layers.append(nn.AvgPool3d((1, 3, 3), stride=cfg.stride, padding=cfg.padding_avg))
+            layers.append(
+                ConvBlock3D(
+                    in_planes=cfg.input_channels,
+                    out_planes=cfg.out_channels,
+                    kernel_size=(1, 1, 1),
+                    padding=(0, 0, 0),
+                    norm_layer=norm_layer,
+                    activation_layer=nn.Identity,
+                    conv_type=conv_type,
+                    tf_like=tf_like,
+                ),
+            )
+            self.res = nn.Sequential(*layers)
+        # ReZero
+        self.alpha = nn.Parameter(torch.tensor(0.0), requires_grad=True)
+
+    def forward(self, x: Tensor) -> Tensor:
+        """Forward function of BasicBneck."""
+        residual = self.res(x) if self.res is not None else x
+        if hasattr(self, "expand"):
+            x = self.expand(x)
+        x = self.deep(x)
+        x = self.se(x)
+        x = self.project(x)
+        return residual + self.alpha * x
+
+
+class MoViNet(nn.Module):
+    """MoViNet class used for video classification."""
+
+    def __init__(
+        self,
+        cfg: Config,
+        conv_type: str | None = "3d",
+        tf_like: bool | None = False,
+    ) -> None:
```

Comment:
This could fit into single line?
```suggestion
    def __init__(self, cfg: Config, conv_type: str | None = "3d", tf_like: bool | None = False) -> None:
```

Reply:
Done. https://github.com/openvinotoolkit/training_extensions/pull/2627/commits/7169459c0e1c71d2ee480b906c4533af36004ca9